changeset:   313:e59e807a91a8
tag:         tip
user:        Pierre Jaury <pierre@jaury.eu>
date:        Mon Sep 23 10:46:53 2013 +0200
summary:     Fix #85 regarding backward compatibility with python-keyring <= 3.0

diff -r bae6c45073bd -r e59e807a91a8 wheel/tool/__init__.py
--- a/wheel/tool/__init__.py	Sun Sep 15 18:08:42 2013 -0400
+++ b/wheel/tool/__init__.py	Mon Sep 23 10:46:53 2013 +0200
@@ -47,7 +47,7 @@
     kr = keyring.get_keyring()
     kr.set_password("wheel", vk, sk)
     sys.stdout.write("Created Ed25519 keypair with vk={0}\n".format(vk))
-    if isinstance(kr, keyring.backend.BasicFileKeyring):
+    if isinstance(kr, keyring.backends.file.BaseKeyring):
         sys.stdout.write("in {0}\n".format(kr.file_path))
     else:
         sys.stdout.write("in %r\n" % kr.__class__)

